WeatherUI/Views/CityWeatherView/CityWeatherView.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  CityWeatherView.swift
+//  WeatherUI
+//
+//  Created by kostya on 04.07.2022.
+//
+
+import SwiftUI
+
+struct CityWeatherView: View {
+    let model: CityWeather
+
+    var body: some View {
+        ScrollView {
+            VStack(alignment: .leading, spacing: 0.0) {
+                HStack {
+                    Text(model.cityName)
+                        .font(.system(size: 18, weight: .regular))
+                    Image("city_pin")
+                }
+                HStack(alignment: .center) {
+                    VStack(alignment: .leading, spacing: 8.0) {
+                        Text(model.currentTemperature.stringValue)
+                            .font(.system(size: 64, weight: .regular))
+                        Group {
+                            Text(model.conditions.description)
+                            Text(model.date)
+                        }
+                        .font(.system(size: 18, weight: .regular))
+                    }
+                    Spacer()
+                    Image(model.conditions.bigImageName)
+                }
+                HStack {
+                    Label(model.windSpeed.stringValue, image: "wind")
+                    Spacer()
+                    Label("\(model.humidity) %", image: "humidity")
+                    Spacer()
+                    Label("\(String(format: "%.3f", model.pressure)) mBar", image: "pressure")
```

Comment:
Should we got the value already with the unit text here?

Reply:
updated along with using Measurement instead of custom units entities